nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/BufferingServletOutputStream.java

```diff
@@ -269,4 +270,18 @@ public static void stopBuffering(OutputStream out) throws IOException {
         }
     }
 
+    @Override
+    public boolean isReady() {
+        if (streaming) {
```

Comment:
Change this into a Ternary ?

Reply:
I'm keeping the same style as other tests of `streaming` in this class. It also helps to add comments.

Reply:
Fair enough. 👍